### PR TITLE
Add ability to select `title` element of SVG with `getByTitle`

### LINKED ISSUE #83: Add support for `<title>` element in SVG with `getByTitle`
### Describe the feature you'd like:

There is a `<title>` element that can be rendered within an SVG element, see [MDN](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/title), and would be nice to be considered when doing a `getByTitle` assertion. Currently it does work with `getByText`, but that might be incorrect considering it behaves more like a title attribute than text content.

If accepted, I will work up a PR adding support for it.

### Suggested implementation:

Would need to find the `<title>` element and ensure it's parent is a `<svg>` element and allow that to be included in the assertion.

### Describe alternatives you've considered:

We currently use `getByText` as a selector.

### PR DESCRIPTION
**Pull request by wyze**
Close #83

**What**: Add ability to select `title` element of SVG with `getByTitle`

**Why**: The `title` element of an SVG behaves the same as a `title` attribute on other elements, so the `getByTitle` query should find that element.

**How**: Update `queryAllByTitle` function to check for `[title]` or `svg > title` selectors and use either one that is returned for the match.

**Checklist**:

- [x] Documentation
- [x] Tests
- [x] Ready to be merged
- [x] Added myself to contributors table

### CODE REVIEW

File: src/queries.js [hunk 1]
```diff
@@ -82,12 +82,28 @@ function queryByText(...args) {
   return firstResultOrNull(queryAllByText, ...args)
 }
 
+function queryAllByTitle(
+  container,
+  text,
+  {exact = true, collapseWhitespace = true, trim = true} = {},
+) {
+  const matcher = exact ? matches : fuzzyMatches
```
Comment (kentcdodds) [inline src/queries.js]: I'm guessing this is where we're missing some coverage. Could you add one more assertion that specifies `exact: false` to cover this?

Comment (wyze) [reply src/queries.js]: Sure thing. Will get that pushed up shortly.

**Comment by kentcdodds** [comment]
Thanks so much for your help! I've added you as a collaborator on the project. Please make sure that you review the `other/MAINTAINING.md` and `CONTRIBUTING.md` files (specifically the bit about the commit messages and the git hooks) and familiarize yourself with the code of conduct (we're using [the contributor covenant](http://contributor-covenant.org/)). You might also want to [watch the repo](https://help.github.com/articles/watching-repositories/) to be notified when someone files an issue/PR. Please continue to make PRs as you feel the need (you can make your branches directly on the repo rather than your fork if you want). Thanks! And welcome to the team :)

**Pull request merged**